config/database.yml.test

```diff
@@ -1,12 +0,0 @@
-development:
```

Comment:
Ah, understood. There shouldn't be any changes to this file in the development branch.